web/src/bootstrap_typeahead.ts

```diff
@@ -658,6 +660,10 @@ export class Typeahead<ItemType extends string | object> {
                 ) {
                     return;
                 }
+                if (pseudo_keycode === 8) {
+                    this.lookup(this.hideOnEmptyAfterBackspace);
+                    return;
+                }
```

Comment:
I think you want this to happen right before the other call to lookup, so the `openInputFieldOnKeyUp` can happen

Reply:
Done.